client/src/components/EmailModal/EmailModal.component.tsx

```diff
@@ -0,0 +1,146 @@
+  const { errors: formErrors } = formState
+  const [isLoading, setIsLoading] = useState<boolean>(false)
+
+  const onSubmit: SubmitHandler<Enquiry> = async (data) => {
+    setIsLoading(true)
+    await onConfirm(data)
+    setIsLoading(false)
+    reset()
+    onClose()
+  }
+
+  return (
+    <Modal isOpen={isOpen} onClose={onClose} size="xl">
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <ModalOverlay />
+        <ModalContent w="660px">
+          <ModalHeader>
+            <Text textStyle="h2" color="secondary.700">
+              {`${agency.shortname.toUpperCase()} Enquiry Form`}
+            </Text>
+          </ModalHeader>
+          <ModalCloseButton />
+          <ModalBody>
+            <VStack align="left" spacing={0}>
+              <Text>
+                {`This enquiry form will be sent to ${agency.longname}. Please do
+                note that we would take at least 3 working days to process your
+                enquiry. Thank you.`}
+              </Text>
+              <Box h={4} />
+              <Text textStyle="subhead-1" color="secondary.700">
+                Question Title
+              </Text>
+              <Box h={3} />
+              <Input
+                focusBorderColor="primary.500"
+                errorBorderColor="error.500"
+                isInvalid={formErrors.questionTitle}
+                {...register('questionTitle', {
+                  required: true,
+                })}
+              />
+              {formErrors.questionTitle && errorLabel('This field is required')}
+              <Box h={4} />
+              <Text textStyle="subhead-1" color="secondary.700">
+                Description
+              </Text>
+              <Box h={3} />
+              <Textarea
+                focusBorderColor="primary.500"
+                errorBorderColor="error.500"
+                isInvalid={formErrors.description}
+                h="144px"
+                {...register('description', {
+                  required: true,
+                })}
+              />
+              {formErrors.description && errorLabel('This field is required')}
+              {/* <Box h={4} />
+              <Text textStyle="subhead-1" color="secondary.700">
+                Sender email
+              </Text>
+              <Text textStyle="body-2" color="secondary.400">
+                Please fill in a valid email so that we can get back to you
+              </Text>
+              <Box h={3} />
+              <Input
+                focusBorderColor="primary.500"
+                errorBorderColor="error.500"
+                isInvalid={formErrors.senderEmail}
+                placeholder="jony_tan@gmail.com"
+                {...register('senderEmail', {
+                  required: true,
+                  pattern: /^\S+@\S+$/i,
+                })}
+              />
+              {formErrors.senderEmail &&
+                errorLabel('Please enter a valid email')} */}
```

Comment:
Is this being put in place ahead of time?

Reply:
Yes, since the email is currently sent from the user, this field is not needed. Its for the next iteration when we send from askgov but loop the user in